### PR TITLE
[fix bug 1269059] /contribute/friends updates.

### PR DESCRIPTION
**Pull request by jpetto**
## Description

Update copy (and subsequently, design and UX) on `/contribute/friends` page.

## Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1269059

## Testing

Test on major & minor browsers with JS on and off.

Note that this page is not localized and is intended only for en-US audiences (for now, anyway).

## Checklist
- [x] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/contribute/friends.html [hunk 1]
```diff
@@ -59,6 +78,14 @@
         <div class="field field-country">{{ newsletter_form.country|safe }}</div>
         <div class="field field-format">{{ newsletter_form.fmt|safe }}</div>
         <div class="field field-privacy">{{ newsletter_form.privacy|safe }}</div>
+        <div class="field field-fx-and-you">
+          <label for="id_fx-and-you">
+            <input id="id_fx-and-you" name="fx-and-you" type="checkbox">
+            <span class="title">
+              {{ _('Yes, I’d like to stay up-to-date with all the latest and greatest Firefox news by subscribing to the Firefox + You newsletter here.') }}
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/contribute/friends.html]: Nit: Worth placing some emphasis on the name of the newsletter in this line of copy? `<em>Firefox + You</em>`

File: tests/functional/contribute/test_friends.py [hunk 2]
```diff
@@ -11,9 +11,7 @@
 @pytest.mark.smoke
 @pytest.mark.nondestructive
 def test_signup_default_values(base_url, selenium):
-    page = ContributeFriendsPage(selenium, base_url).open()
-    page.click_show_signup_form()
-    assert page.is_signup_form_displayed
+    page = ContributeFriendsPage(base_url, selenium).open()
```
Comment (alexgibson) [inline tests/functional/contribute/test_friends.py]: The order of params here should be `selenium, base_url` (this changed just recently when we moved to PyPOM). Same applies to the tests below.

File: tests/pages/contribute/friends.py [hunk 3]
```diff
@@ -46,6 +47,10 @@ def is_privacy_policy_link_displayed(self):
         return self.is_element_displayed(*self._privacy_policy_link_locator)
 
     @property
+    def newsletters(self):
+        return self.find_element(self._newsletters_locator).get_attribute('value')
```
Comment (alexgibson) [inline tests/pages/contribute/friends.py]: `*self._newsletters_locator`

File: tests/pages/contribute/friends.py [hunk 4]
```diff
@@ -58,17 +63,18 @@ def sign_up_successful(self):
     def text_format_selected(self):
         return self.find_element(*self._text_format_locator).is_selected()
 
+    def accept_fx_and_you(self):
+        el = self.find_element(self._fx_and_you_locator)
```
Comment (alexgibson) [inline tests/pages/contribute/friends.py]: `*self._fx_and_you_locator`

**Comment by alexgibson** [comment]
A few fix-ups to get the tests passing again after rebase, but otherwise this looks great. Tested in most major browsers and also tested in FF with JS disabled. r+wc 🍟

**Comment by jpetto** [comment]
Good suggestion on emphasizing the newsletter name. Fixed up the tests as well. Thanks!

**Comment by alexgibson** [comment]
r+ 🐧

**Pull request merged**